packages/@lightningjs/ui-components/src/components/CardContent/CardContentVerticalSmall.stories.js

```diff
@@ -50,21 +49,24 @@ export const CardContentVerticalSmall = args =>
 
 CardContentVerticalSmall.storyName = 'CardContentVerticalSmall';
 
+//Creating a shallow copy of CardContent.args object and removing metadata details property from cardContentArgs
+const cardContentArgs = { ...CardContent.args };
+delete cardContentArgs.metadata_details;
+
 CardContentVerticalSmall.args = {
   mode: 'focused',
   ...CardContent.args
 };
+
+//Creating a shallow copy of CardContent.argTypes object and removing metadata details property from cardContentArgsTypes
+const cardContentArgsTypes = { ...CardContent.argTypes };
+delete cardContentArgsTypes.metadata_details;
+
 CardContentVerticalSmall.argTypes = {
   ...createModeControl({ summaryValue: CardContentVerticalSmall.args.mode }),
-  shouldCollapse: {
-    control: 'boolean',
-    description: controlDescriptions.shouldCollapse,
-    table: {
-      defaultValue: { summary: false }
-    }
-  },
   ...CardContent.argTypes
```

Comment:
Same as above? It looks like you're creating copies base off of `CardContent` but not using them?